contracts/AirbroFactorySMCampaign.sol

```diff
@@ -0,0 +1,132 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.14;
+
+import "./airdropsSMCampaign/TokenDropSMCampaign.sol";
+import "./airdropsSMCampaign/ExistingTokenDropSMCampaign.sol";
+import "./airdropsSMCampaign/Existing1155NftDropSMCampaign.sol";
+import "./Airbro1155Contract.sol";
+
+
+/// @title Airbro - NFT airdrop tool factory contract - for owners of 1155 Nfts
+contract AirbroFactorySMCampaign {
+    // index of deployed airdrop contracts
+    address[] public airdrops;
+    uint256 public totalAirdropsCount = 0;
+    address public admin = 0xF4b5bFB92dD4E6D529476bCab28A65bb6B32EFb3;
```

Comment:
this looks like a constant to me / immutable

Reply:
There is a method with which the admin variable (a.k.a the backend wallet address) can be changed.

https://github.com/MVPWorkshop/Airbro-contracts/blob/4584366e9c143f8564c7e41f6cd0233c6acfa187/contracts/AirbroFactorySMCampaign.sol#L126-L131

Reply:
On the bottom of the contract we have a method changeAdmin to change the admin if needed, so I would keep it muttable @bxmmm1